### PR TITLE
Ests 53953

### PR DESCRIPTION
**Pull request by silvam11**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]
- [ ] This is a bug
- [ ] This is a feature request
- [ ] This is a syntax/style fix
- [ ] This is an improvement to code quality

---

Provide a clear description of changes:
 
[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/adapter/ValidateStoragePoolResponseAdapter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.dell.cpsd.paqx.dne.service.amqp.adapter;
```
Comment (giddyelysium) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/adapter/ValidateStoragePoolResponseAdapter.java]: The Copyright or Confidentiality statement is incorrect; please review;

- "Copyright © [date] Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Copyright &copy; 2017 Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Dell EMC Confidential/Proprietary Information"

See [our documentation][contributing] for details on how to contribute.

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandler.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class implements the logic to find Storage pool entries
+ * <p>
```
Comment (giddyelysium) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandler.java]: JavaDoc Error may occur here due to opening paragraph and no closing

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/transformers/StoragePoolEssRequestTransformer.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package com.dell.cpsd.paqx.dne.transformers;
```
Comment (giddyelysium) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/transformers/StoragePoolEssRequestTransformer.java]: The Copyright or Confidentiality statement is incorrect; please review;

- "Copyright © [date] Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Copyright &copy; 2017 Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Dell EMC Confidential/Proprietary Information"

See [our documentation][contributing] for details on how to contribute.

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

File: dne-paqx/src/test/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
+package com.dell.cpsd.paqx.dne.service.task.handler.preprocess;
```
Comment (giddyelysium) [inline dne-paqx/src/test/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandlerTest.java]: The Copyright or Confidentiality statement is incorrect; please review;

- "Copyright © [date] Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Copyright &copy; 2017 Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Dell EMC Confidential/Proprietary Information"

See [our documentation][contributing] for details on how to contribute.

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

File: dne-paqx/src/test/java/com/dell/cpsd/paqx/dne/transformers/StoragePoolEssRequestTransformerTest.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
+package com.dell.cpsd.paqx.dne.transformers;
```
Comment (giddyelysium) [inline dne-paqx/src/test/java/com/dell/cpsd/paqx/dne/transformers/StoragePoolEssRequestTransformerTest.java]: The Copyright or Confidentiality statement is incorrect; please review;

- "Copyright © [date] Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Copyright &copy; 2017 Dell Inc. or its subsidiaries.  All Rights Reserved."
- "Dell EMC Confidential/Proprietary Information"

See [our documentation][contributing] for details on how to contribute.

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/adapter/ValidateStoragePoolResponseAdapter.java (hunk 1 above)
Comment (silvam11) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/amqp/adapter/ValidateStoragePoolResponseAdapter.java]: thanks!

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandler.java (hunk 2 above)
Comment (silvam11) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindScaleIOTaskHandler.java]: thanks

**Pull request merged**